### ISSUE TITLE
Rollup: 'InfiniteScrollModule' is not exported by node_modules/angular2-infinite-scroll/angular2-infinite-scroll.js

### ISSUE THREAD

**Issue by EgorBurykin** [orizens/angular2-infinite-scroll#92]
Hello, i'am not sure is it issue of angular2-infinite-scroll (0.2.6) itself or rollup-plugin-commonjs(5.0.5). I just can't make Rollup after AOT compilation work. Code rollup.js:

```
plugins: [
    nodeResolve({jsnext: true, module: true}),
    commonjs({
        include: ['node_modules/rxjs/**','node_modules/angular2-infinite-scroll/src/**'],
        namedExports: { 
            'node_modules/angular2-infinite-scroll/angular2-infinite-scroll.js': [ 'InfiniteScrollModule' ]
        }
    }),
    uglify()
]
```

And in my module:

```
import { InfiniteScrollModule } from 'angular2-infinite-scroll/angular2-infinite-scroll';
```

And in system.config.js:

```
map: {
//...
    'angular2-infinite-scroll': 'npm:angular2-infinite-scroll'
}
```

**Comment by orizens** [comment]
hi.
can you supply a sample repo for that?

**Comment by EgorBurykin** [comment]
Sure. [https://github.com/EgorBurykin/angular2-infinite-scroll-problem](url)

**Comment by EgorBurykin** [comment]
Any news here? Is there any way to make it work? Any help would be great)

**Comment by orizens** [comment]
hi.
I didn't identify the issue yet.

**Comment by orizens** [comment]
hi @EgorBurykin 
please try beta version ```angular2-infinite-scroll@0.2.8-0``` before I move this package to the next patch version.

**Comment by orizens** [comment]
fixed by 252ba10199f279366c00e69dfdddf36bd2e1adf1
